packages/@ember/-internals/glimmer/lib/component-managers/internal.ts

```diff
@@ -41,37 +41,31 @@ export interface InternalComponentState {
 }
 
 export class InternalComponentDefinition
```

Comment:
Calling these interfaces "internal" now seems odd (since glimmer-vm also has a set of things it calls "internal"). Not sure what the best name here would be, but something like `EmberInternal*` seems better than the current confusion (at least _my_ confusion 😜 ).

Reply:
This is something that I plan on addressing in the next set of PRs, which will upstream most of the component manager logic. Part of that will be unifying internal and public component managers so they can both be used with `setComponentManager`, and in general use the same infrastructure. This way for instance, curly components will be able to be used in strict mode, since their internal manager will be associated with `setComponentTemplate`. 

As part of that work, this manager will likely be renamed `InputComponentManager` or something like that. I could rename it as part of this PR I suppose, it's just a bit weird since the resolver in Ember still has a notion of "internal", and I didn't want to get rid of that all at once either.